tide-whisperer.go

```diff
@@ -60,6 +64,21 @@ var (
 	errorInvalidParameters = detailedError{Status: http.StatusInternalServerError, Code: "invalid_parameters", Message: "one or more parameters are invalid"}
 
 	storage store.Storage
+
+	tideWhispererConfig = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
```suggestion
	tideWhispererConfigValid = promauto.NewGauge(prometheus.GaugeOpts{
```
Rename variable for consistency.

Reply:
:+1: